modules/virt-creating-custom-resources-hpp.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-custom-resources-hpp_{context}"]
+= Creating host path provisioner custom resources
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, the system will:
+* Update the legacy host path provisioner (HPP)
```

Comment:
In the rendered output this is not showing up as a bullet list, which I am guessing it should be.

Reply:
Thanks, got it.